### PR TITLE
Supernova small UI fix til

### PR DESCRIPTION
**Pull request by TilmanHaupt**
fixes: [#487](https://github.com/sapcc/juno/issues/487)

New alphabetical sorting in the select and new error messages for supernova.

Bildschirmfoto 2024-01-23 um 16 16 54
Bildschirmfoto 2024-01-23 um 16 18 20
Bildschirmfoto 2024-01-23 um 16 18 38

### CODE REVIEW

File: apps/supernova/src/components/filters/FilterSelect.jsx [hunk 1]
```diff
@@ -84,13 +84,17 @@ const FilterSelect = () => {
...
         >
-          {filterLabels?.map((filter) => (
-            <SelectOption
-              value={filter}
-              label={humanizeString(filter)}
-              key={filter}
-            />
-          ))}
+          {filterLabels &&
+            [].slice
```
Comment (edda) [inline apps/supernova/src/components/filters/FilterSelect.jsx:88]: Why not just do `filterLabels?.sort().map((filter)) => ...` ?
What advantage does the `[].slice.call(filterLabels)` give?

**Review by edda — CHANGES_REQUESTED** [review]

File: apps/supernova/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "supernova",
-  "version": "0.9.6",
+  "version": "0.9.7",
```
Comment (ArtieReus) [inline apps/supernova/package.json:3]: Did you also run the `npm install -w supernova` to update the lock file?

**Review by ArtieReus — CHANGES_REQUESTED** [review]

**Review by ArtieReus — APPROVED** [review]
lgtm

File: apps/supernova/src/AppContent.jsx [hunk 3]
```diff
@@ -70,19 +70,33 @@ const AppContent = (props) => {
...
     }
 
-    addMessage({
-      variant: "error",
-      text: parseError(alertsError),
-    })
+    if (parseError(alertsError) === "Failed to fetch") {
+      addMessage({
+        variant: "error",
+        text: "Sorry, there was an issue fetching the data. Possible causes could include network issues, incorrect URL, or server outages. ",
```
Comment (ArtieReus) [inline apps/supernova/src/AppContent.jsx:76]: Would you add the message `"Sorry, there was an issue fetching the data. Possible causes could include network issues, incorrect URL, or server outages.` to the parseError method so it is not duplicated in the code?

**Review by ArtieReus — CHANGES_REQUESTED** [review]

**Review by ArtieReus — APPROVED** [review]
lgtm

**Pull request merged**